benchopt/cli/helpers.py

```diff
@@ -62,19 +62,27 @@ def print_info(cls_list, env_name=None):
     for cls in cls_list:
         print(f"## {cls.name}")
         # doc
-        if hasattr(cls, '__doc__') and cls.__doc__ is not None and \
-                len(cls.__doc__) > 0:
+        if hasattr(cls, '__doc__') and cls.__doc__:
             print(f"> doc: {cls.__doc__}")
         # parameters
         if hasattr(cls, 'parameters') and cls.parameters:
             print("> parameters:")
             for param, value in cls.parameters.items():
                 values = '\', \''.join(map(str, value))
-                print(f"    - '{param}': '{values}'")
+                print(f"    '{param}': '{values}'")
         # install command
-        if hasattr(cls, 'install_cmd') and cls.install_cmd and \
-            hasattr(cls, 'requirements') and cls.requirements:
-            print(f"> dependencies: {cls.install_cmd} install {' '.join(cls.requirements)}")
+        if hasattr(cls, 'requirements') and cls.requirements:
+            print("> requirements:")
+            packages = cls.requirements
+            pip_packages = [pkg[4:] for pkg in packages \
```

Comment:
I think the continuation sign is not needed since you are inside brackets

Reply:
Yes, you are right. Old habbits...